src/d3-graph.ts

```diff
@@ -225,72 +226,70 @@ export default class D3Graph {
    * @private
    * @return {undefined}
    */
-  _enableClickToCenter() {
-    const that = this
-    this._nodes.on('click', function(d) {
-      const {translation, scale} = that._getGraphTranslationAndScale()
+  private _enableClickToCenter(): void {
+    this._nodes.on('click', (d: any) => {
+      const {translation, scale} = this._getGraphTranslationAndScale()
       const x = translation[0] - d.x
       const y = translation[1] - d.y
-      that._g
+      this._g
         .transition()
-        .duration(that._TRANSITION_DURATION)
+        .duration(D3Graph._TRANSITION_DURATION)
         .call(
-          that._zoomHandler.transform,
+          this._zoomHandler.transform,
           d3.zoomIdentity.translate(x, y).scale(scale)
         )
     })
   }
 
-  _enableKeyboardPanning() {
+  private _enableKeyboardPanning(): void {
```

Comment:
A comment should be added here to describe this function, similar to render() and _renderNodes().